drv/sidecar-seq-server/src/front_io.rs

```diff
@@ -8,39 +8,41 @@ use drv_sidecar_front_io::{controller::FrontIOController, phy_smi::PhySmi};
 
 #[allow(dead_code)]
 pub(crate) struct FrontIOBoard {
-    pub fruid: I2cDevice,
```

Comment:
Why remove `fruid` and `i2c_task` here?  (they're here as a small caching optimization, so we don't need to look them up repeatedly)`

Reply:
Because they're only used in the `FrontIOBoard::present(..)` function which, unless a front IO board power cycle is requested, is only called once on task init (through `ServerImpl::front_io_board_preinit`). I figured I'd let the lookup happen then rather than on struct creation. And when the `fruid` handle is used we're interacting with the EEPROM over I2C, which I figured dwarfs any speedups we'd get from keeping this handle around.

But I did realize that during sequencer `main` we do get the task handle repeatedly, so there it makes sense to do it once and use it several times.

Reply:
Gotcha, I don't feel too strongly about this